### PR TITLE
Add 'standing: playerinfo-box' component

### LINKED ISSUE #2: Add 'standing: playerinfo-box' component

### PR DESCRIPTION
**Pull request by JuKahl**
- 🔨 Add playerinfo-picture
- 🔨 Add playerinfo-pick
- 🔨 Add playerinfo-box
- 💄 Add styling

[Deployment](https://king-of-the-playerinfo-ct3qstz.herokuapp.com/storybook/index.html?path=/story/common-button--primary)

### CODE REVIEW

File: components/standing/Playerinfo.module.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.playerinfoBox {
```
Comment (RonjaMeinecke) [inline components/standing/Playerinfo.module.css]:
```suggestion
.playerInfoBox {
```

File: components/standing/Playerinfo.module.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  grid-template-areas: "picture username userpick";
+  width: 22.5rem;
+  height: 4.5rem;
+  border-radius: 20px;
+  color: var(--primary-color);
+  background-color: var(--secondary-color);
+  opacity: 50%;
+}
+
+.playerinfoPicture {
```
Comment (RonjaMeinecke) [inline components/standing/Playerinfo.module.css]:
```suggestion
.playerInfoPicture {
```

File: components/standing/Playerinfo.module.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  border-radius: 100%;
+  border: 0.1em solid white;
+  justify-self: center;
+  align-self: center;
+  object-fit: cover;
+  height: 3.125rem;
+  width: 3.125rem;
+}
+
+.playerinfoUsername {
```
Comment (RonjaMeinecke) [inline components/standing/Playerinfo.module.css]:
```suggestion
.playerInfoUsername {
```

File: components/standing/Playerinfo.module.css [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  grid-area: username;
+  font-size: 1.25rem;
+  justify-self: center;
+  align-self: center;
+}
+
+.playerinfoPick {
```
Comment (RonjaMeinecke) [inline components/standing/Playerinfo.module.css]:
```suggestion
.playerInfoPick {
```

File: components/standing/Playerinfo.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import { Story, Meta } from "@storybook/react/types-6-0";
+import PlayerinfoBox, { PlayerinfoProps } from "./PlayerinfoBox";
+
+export default {
+  title: "Common/PlayerinfoBox",
+  component: PlayerinfoBox,
+  parameters: { layout: "fullscreen" },
+} as Meta;
+
+const Template: Story<PlayerinfoProps> = (args) => <PlayerinfoBox {...args} />;
```
Comment (RonjaMeinecke) [inline components/standing/Playerinfo.stories.tsx]:
```suggestion
const Template: Story<PlayerinfoProps> = (args) => <PlayerInfoBox {...args} />;
```

File: components/standing/PlayerinfoBox.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import styles from "./Playerinfo.module.css";
+import UserPicture from "./PlayerinfoPicture";
+import UserPick from "./PlayerinfoPick";
+
+export type PlayerinfoProps = {
```
Comment (RonjaMeinecke) [inline components/standing/PlayerinfoBox.tsx]:
```suggestion
export type PlayeInfoProps = {
```

File: components/standing/PlayerinfoBox.tsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  picture: string;
+  username: string;
+  pick: string;
+  pickLogo: string;
+  opponent: string;
+  opponentLogo: string;
+};
+
+export default function PlayerinfoBox({
```
Comment (RonjaMeinecke) [inline components/standing/PlayerinfoBox.tsx]:
```suggestion
export default function PlayerInfoBox({
```

**Review by DNaunin — APPROVED** [review]
LGTM! 🚀

**Review by RonjaMeinecke — APPROVED** [review]
stark 🦾

File: components/standing/PlayerinfoBox.tsx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  picture,
+  username,
+  pick,
+  pickLogo,
+  opponent,
+  opponentLogo,
+}: PlayerinfoProps) {
```
Comment (phlgr) [inline components/standing/PlayerinfoBox.tsx]:
```suggestion
}: PlayerInfoProps) {
```

File: components/standing/PlayerinfoBox.tsx [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <section className={`${styles.playerinfoBox}`}>
+      <UserPicture picture={picture} />
+      <p className={`${styles.playerinfoUsername}`}>{username}</p>
+      <UserPick
```
Comment (phlgr) [inline components/standing/PlayerinfoBox.tsx]: This style naming is not how it is supposed to be with `module.css`-files. 
Take a look at the `Use CSS Modules in React` Paragraph: 
https://www.triplet.fi/blog/practical-guide-to-react-and-css-modules/

**Pull request merged**